pkg/api/frontendsettings.go

```diff
@@ -457,6 +457,10 @@ func (hs *HTTPServer) pluginSettings(ctx context.Context, orgID int64) (map[stri
 			OrgID:    orgID,
 			Enabled:  true,
 		}
+		// override if this is an installed, but inactive secretsmanager plugin
```

Comment:
**Thought:** maybe we should some comments on why this is needed? (Or maybe the current comment already tells it and it's just me not understanding?)

Reply:
it is basically for the reasons described at the top of the PR. I will add another comment that explains it more clearly. Thanks for the review!